units/src/fee_rate/mod.rs

```diff
@@ -15,17 +15,42 @@ mod encapsulate {
     ///
     /// This is an integer newtype representing fee rate in `sat/kwu`. It provides protection
     /// against mixing up the types as well as basic formatting features.
+    ///
+    /// We reserve the right post-1.0 to play around with precision.
     #[derive(Debug, Copy, Clone, Eq, PartialEq, Ord, PartialOrd, Hash)]
-    pub struct FeeRate(u64);
+    pub struct FeeRate {
+        /// The fee rat in sats per kwu.
+        pub(crate) sat_per_kwu: u64,
+        /// Some if this fee rate was created using `from_sat_per_kvb`.
```

Comment:
Why not `sat_per_vb`?  This would allow for code to easily align with bitcoin core since the sat_per_vb is the default.  sat_per_kvb isn't standard AFAIK.